src/CalculatorUITestFramework/StandardCalculatorPage.cs

```diff
@@ -45,5 +60,177 @@ public string GetCalculatorResultText()
         {
             return this.CalculatorResult.Text.Replace("Display is", string.Empty).Trim();
         }
+        /// <summary>
+        /// Gets the text from the display control in KOT mode and removes the narrator text that is not displayed in the UI.
+        /// </summary>
+        /// <returns>The string shown in the UI.</returns>
+        public string GetKOTCalculatorResultText()
+        {
+            return this.CalculatorAlwaysOnTopResults.Text.Replace("Display is", string.Empty).Trim();
+        }
+        /// <summary>
+        /// Gets the text from the Header
+        /// </summary>
+        /// <returns>The string shown in the UI.</returns>
+        public string GetCalculatorHeaderText()
+        {
+            return this.Header.Text;
+        }
+        /// <summary>
+        /// Gets the text from the Calculator Expression control and removes narrator text that is not displayed in the UI.
+        /// </summary>
+        /// <returns>The string shown in the UI.</returns>
+        public string GetCalculatorExpressionText()
+        {
+            return this.CalculatorExpression.Text.Replace("Expression is", string.Empty).Trim();
+        }
+        public void StandardNonKoTMode()
+        {
+            string sourceF = this.session.PageSource;
+            if (sourceF.Contains("ExitAlwaysOnTopButton"))
+            {
+                this.ExitAlwaysOnTopButton.Click();
+                Assert.AreEqual("Standard", this.GetCalculatorHeaderText());
+            }
+            else
+            {
+                string source2 = this.session.PageSource;
+                if (source2.Contains("NormalAlwaysOnTopButton"))
+                {
+                    return;
+                }
+                else
+                {
+                    throw new NotFoundException("Could not find the 'Keep on top' button");
+                }
+            }
+        }
+        ///// <summary>
+        ///// Ensures the calculator is in KOT(Keep on top) mode, and verifies that the KOT mode through the absence of the header
+        ///// </summary>
+        public void StandardKoTMode()
+        {
+            string sourceG = this.session.PageSource;
+            if (sourceG.Contains("NormalAlwaysOnTopButton"))
+            {
+                this.EnterAlwaysOnTopButton.Click();
+                string sourceH = WinAppDriver.Instance.CalculatorSession.PageSource;
+                if (sourceH.Contains("Header"))
+                {
+                    throw new NotFoundException("Failed to enter 'Keep on top' mode; In KoT mode, Calculator does not have header");
+                }
+                else
```

Comment:
You can remove all of the else blocks and just let the method end.

Reply:
Not all of them, but the ones that only have "return" in them.

Reply:
The changes will be in my next commit